src/ai/backend/manager/services/model_serving/services/model_serving.py

```diff
@@ -634,18 +634,22 @@ async def generate_token(self, action: GenerateTokenAction) -> GenerateTokenActi
             wsproxy_api_token = sgroup["wsproxy_api_token"]
 
         await verify_user_access_scopes(self._db, action.requester_ctx, endpoint.session_owner)
-
         body = {"user_uuid": str(endpoint.session_owner), "exp": action.expires_at}
         async with aiohttp.ClientSession() as session:
             async with session.post(
                 f"{wsproxy_addr}/v2/endpoints/{endpoint.id}/token",
                 json=body,
                 headers={
+                    "accept": "application/json",
                     "X-BackendAI-Token": wsproxy_api_token,
                 },
             ) as resp:
-                token_json = await resp.json()
-                token = token_json["token"]
+                resp_json = await resp.json()
+                if resp.status != HTTPStatus.OK:
```

Comment:
This doesn't need to be changed in the PR, but it seems better for the response to a create request to return `201`

Reply:
It might be a good idea to check up to `201`